src/main/java/com/eternalcode/parcellockers/user/UserRepository.java

```diff
@@ -0,0 +1,21 @@
+package com.eternalcode.parcellockers.user;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+public interface UserRepository {
+
+    CompletableFuture<Void> save(User user);
+
+    CompletableFuture<Optional<User>> findByName(String name);
+
+    CompletableFuture<Optional<User>> findByUuid(UUID uuid);
+
+    CompletableFuture<List<User>> findAll();
+
+    CompletableFuture<Void> remove(User user);
```

Comment:
to samo co w przypadku obiektu `Parcel` oraz `ParcelLocker`

Reply:
Rollczi wypowiedział się na ten temat i wraz z nim mam takie samo zdanie, obie metody z parametrem zarówno jako obiekt, jak i samo UUID są użyteczne.